ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/SessionProperties.java

```diff
@@ -36,6 +37,27 @@
   private final boolean internalRequest;
   private final Map<String, String> sessionVariables;
 
+  /**
+   * @param mutableScopedProperties   The streamsProperties of the incoming request
+   * @param ksqlHostInfo              The ksqlHostInfo of the server that handles the request
+   * @param localUrl                  The url of the server that handles the request
+   * @param internalRequest           Flag indicating if request is from within the KSQL cluster
+   * @param sessionVariables          Initial session variables
+   */
+  public SessionProperties(
```

Comment:
Is this used?

Reply:
Yup, https://github.com/confluentinc/ksql/blob/0c34efeb57d37ae51b4241f64cfad5e1cb2ea6ea/ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java#L283